### ISSUE TITLE
HelperError: 'scheming_get_dataset_schema' has not been defined

### ISSUE THREAD

**Issue by Civity-NL** [ckan/ckanext-scheming#183]
Hi, 

I have been using ckanext-scheming in 2.7.4 CKAN without problems and last week I upgraded to the 2.8 version and I started getting the following error:
The page will load with a "Server Error" and in the logs there is "HelperError: Helper 'scheming_get_dataset_schema' has not been defined."

The error is exactly the same as this one posted by @amercader [here](https://gist.github.com/amercader/42ac78da709cd7a7fe7a7e99a9771e6a)

The behavior is completely random as far as I can percieve. If I press F5 repeatedly there is +- 50% chance of the page loading correctly or the page giving the Error.

I already tried changing the order of the plugins to check if that has any impact and so far nothing is different.

Might this be related with the these issues? 
- [Issue ckan/4310](https://github.com/ckan/ckan/issues/4310) 
- [Issue ckanext-discovery/1](https://github.com/stadt-karlsruhe/ckanext-discovery/issues/1)

Or
Can it have anything to to with [these lines of code](https://github.com/ckan/ckanext-scheming/blob/master/ckanext/scheming/plugins.py#L74) that by some reason the helpers get unregistered and they need to be registered again but they do not enter the right if clause?

You guys have any suggestion on what can I address this?

**Comment by wardi** [comment]
Thank you for the report! I don't know of anyone using scheming with 2.8 yet so this is useful.

50% success/failure is strange. Do you have an old web process lying around? Are you using mod_wsgi with the wrong configuration like https://github.com/ckan/ckan/issues/4299#issuecomment-401152142 ?

**Comment by Civity-NL** [comment]
Thanks for the quick reply @wardi .
The 50% behavior was caused by some type of caching in my machine. Once I properly tested it with a fresh browser the behavior was consistent, and the result was always obtaining the error.

I was able to stop the error by removing the [following lines](https://github.com/ckan/ckanext-scheming/blob/master/ckanext/scheming/plugins.py#L74-L76) from the code. This goes in line with my suspicion mentioned above, but I don't understand enough the inner-workings of how the helpers are loaded in the Scheming Extensions as well as in CKAN to give a better suggestion.
